commcare_connect/multidb/tests/test_models.py

```
@@ -0,0 +1,38 @@
+    )
+    def test_clean_rejects_invalid_label(self, bad_label):
+        entry = SupersetReplicatedTable(model_label=bad_label)
+        with pytest.raises(ValidationError):
+            entry.clean()
+
+    def test_clean_accepts_valid_label(self):
+        SupersetReplicatedTable(model_label="opportunity.opportunity").clean()
+
+    def test_model_label_must_be_unique(self):
+        SupersetReplicatedTable.objects.create(model_label="opportunity.opportunity")
+        with pytest.raises(Exception):
```

Ruff (0.15.12)

[warning] Do not assert blind exception: `Exception`

(B017)